lib/resque_spec/matchers.rb

```diff
@@ -185,13 +201,13 @@ def schedule_queue_for(actual)
   chain :at do |timestamp|
     @interval = nil
     @time = timestamp
-    @time_info = "at #{@time}"
+    @time_info = " at #{@time}"
   end
 
   chain :in do |interval|
     @time = nil
     @interval = interval
-    @time_info = "in #{@interval} seconds"
+    @time_info = " in #{@interval} seconds"
```

Comment:
I just realized that when there is no `@time_info` two commas will appear as in:

```
expected that Person would have [Les, asdf] scheduled, , but got Person with [Les, Hill] scheduled instead
```

This goes for scenario in https://github.com/leshill/resque_spec/blob/master/spec/resque_spec/matchers_spec.rb#L357 for instance. I think we can 1) merge this as it is (since it's working and is consistent, in my opinion) or 2) I leave it is and you do the small modifications you see are needed when merging (having an extra blank space works better for me than having one extra comma).

What do you think?